test/e2e/framework/test_context.go

```diff
@@ -201,6 +204,7 @@ func RegisterCommonFlags() {
 func RegisterClusterFlags() {
 	flag.BoolVar(&TestContext.VerifyServiceAccount, "e2e-verify-service-account", true, "If true tests will verify the service account before running.")
 	flag.StringVar(&TestContext.KubeConfig, clientcmd.RecommendedConfigPathFlag, os.Getenv(clientcmd.RecommendedConfigPathEnvVar), "Path to kubeconfig containing embedded authinfo.")
+	flag.StringVar(&TestContext.ExternalKubeConfig, fmt.Sprintf("%s-%s", "external", clientcmd.RecommendedConfigPathFlag), "", "Path to kubeconfig containing embedded authinfo for external cluster.")
```

Comment:
s/ExternalKubeConfig/KubemarkExternalKubeConfig/ or something - we need to make it explicit that it's a Kubmark-only thing.